pages/account/manage/event/[[...data]].js

```diff
@@ -349,7 +352,7 @@ export default function ManageEvent({ BASE_URL, event }) {
                       DELETE
                     </Button>
                   )}
-                  <Button type="submit" primary={true}>
+                  <Button type="submit" primary={true} disable={isDisabled}>
```

Comment:
```suggestion
                  <Button type="submit" primary={true} disabled={isDisabled}>
```
button takes ```disabled``` as props not ```disable```